src/documents/tests/test_workflows.py

```diff
@@ -3144,9 +3144,11 @@ def test_workflow_email_consumption_started(self, mock_email_send, mock_post):
 
     @override_settings(
         PAPERLESS_URL="http://localhost:8000",
+        PAPERLESS_FORCE_SCRIPT_NAME="/paperless",
+        BASE_URL="/paperless/",
     )
     @mock.patch("documents.signals.handlers.send_webhook.delay")
-    def test_workflow_webhook_action_body(self, mock_post):
+    def test_workflow_webhook_action_body_with_subpath(self, mock_post):
```

Comment:
Certainly a nit but seems unnecessary to change the test name here, the subpath part isn’t really the “point” of the test (I’m not even sure we really need a test for the change but can’t hurt)